catalyst-gateway/bin/src/event_db/utxo.rs

```diff
@@ -45,9 +74,11 @@ impl EventDB {
                                         .to_string(),
                                 )
                             })?,
-                            &tx.hash().as_slice(), /* temporary until we have foreign key
-                                                    * relationship in the context of
-                                                    * registrations */
+                            &hex::decode(stake_credential_hash.clone()).map_err(|e| {
```

Comment:
Is it possible to take just a reference at this place ?
```Rust
 &hex::decode(&stake_credential_hash)
 ```

Reply:
yes just changed it 👍